modules/metastorage/src/main/java/org/apache/ignite/internal/metastorage/impl/MetaStorageManagerImpl.java

```diff
@@ -200,7 +200,7 @@ public void onDisappeared(ClusterNode member) {
     public void start() {
         this.metaStorageSvcFut = cmgMgr.metaStorageNodes()
                 // use default executor to avoid blocking CMG manager threads
-                .thenComposeAsync(metaStorageNodes -> {
+                .thenCompose(metaStorageNodes -> {
```

Comment:
Is that about 
> The component is used common ForkJoinPool on start

?
There's a comment
> // use default executor to avoid blocking CMG manager threads

So, do you really think that using thenCompose instead of thenComposeAsync won't block CMG manager threads?

Reply:
I am sure, because the code does not execute in CMG thread after all components began start in the common thread pool.

Reply:
The comment about CMG thread are removed from here.